forward/rewriting.lua

```diff
@@ -11,11 +11,7 @@ local rshift = bit.rshift
 
 local godefs = require("godefs")
 
-local L3_IPV4 = 0x0800
-local L3_IPV6 = 0x86DD
-local L4_TCP = 0x06
-local L4_UDP = 0x11
-local L4_GRE = 0x2f
+local networking_magic_numbers = require("networking_magic_numbers")
```

Comment:
Looks like you refactored these constants into their own file, but didn't update the places where they're used?

Reply:
They are globals, so they are used the same way. Would it be better to wrap them in a table to restrict their scope?

Actually oops, the refactoring is completely independent of the bugfix. I'll take that out of the PR.